clusterDeployer.py

```diff
@@ -885,7 +913,7 @@ def boot_iso_x86(self, worker: NodeConfig, iso: str) -> None:
         nfs = NFS(lh, self._cc.external_port)
 
         bmc = host.bmc_from_host_name_or_ip(worker.node, worker.bmc_ip, worker.bmc_user, worker.bmc_password)
-        h = host.HostWithBF2(host_name, bmc)
+        h = Host(host_name, bmc)
```

Comment:
This is still weird. Can you elaborate why we are removing HostWithBF2?

Reply:
that seems to be a mistake. I think I accidentally modified it without meaning to. Apologies